docs/src/main/sphinx/admin/properties-logging.rst

```diff
@@ -56,6 +56,15 @@ rotation replaces old content.
 
 The maximum file size for the general application log file.
 
+``log.compression``
+^^^^^^^^^^^^
```

Comment:
This needs to match the length of the text above

Reply:
Thank you for catching this - this is fixed now.